pom.xml

```diff
@@ -863,7 +863,7 @@
             <goals>clean deploy</goals>
             <preparationGoals>clean package</preparationGoals>
             <tagNameFormat>rel/@{project.version}</tagNameFormat>
-            <releaseProfiles>apache-release,accumulo-release</releaseProfiles>
+            <releaseProfiles>accumulo-release</releaseProfiles>
```

Comment:
I think there are still stuff in the Apache parent POM that would need to be activated with the apache-release profile, so we'd still need both.